langchain/src/callbacks/handlers/tracers.ts

```diff
@@ -55,10 +74,39 @@ export interface ToolRun extends BaseRun {
 
 export type Run = LLMRun | ChainRun | ToolRun;
 
+export interface BaseRunV2 {
+  id: string;
+  name: string;
+  start_time: number;
+  end_time: number;
+  extra: object;
```

Comment:
this should be optional and some version of `Record<string, Any>`

Reply:
@agola11 FYI currently still has to be provided - I'll update the backend to let it be optional

Reply:
sg, my bad for overlooking that